src/test/e2e/29_config_file_test.go

```diff
@@ -38,15 +44,17 @@ func TestConfigFile(t *testing.T) {
 	e2e.CleanFiles(path)
 }
 
-func configFileTests(t *testing.T, dir, path string) {
+func configFileTests(t *testing.T, dir, path string, configPath string) {
```

Comment:
A bit of a nit, but I think it would be simpler if the function signature was
```go
func configFileTests(t *testing.T, dir, path string, extraArgs ...string)
```
Then it could be called like below
```go
configFileTests(t, dir, path, "--config-path", configPath)
```
and the args could simply be appended like this
```go
args := append([]string{"package", "create", dir, "--confirm"}, extraArgs...)
```